src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -133,26 +197,82 @@ def load(key: str):
     uri = f"s3://{S3_BUCKET}/{key}"
     logger = prefect.context.get("logger")
     logger.info(f"Snowflake loading key: {uri}")
-    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(data={'uri': uri}, query=IMPORT_SQL)
+    return SnowflakeQuery(**SNOWFLAKE_DEFAULT_DICT).run(
+        data={"uri": uri}, query=IMPORT_SQL
+    )
 
 
 @task()
 def cleanup(key: str):
     bucket = S3_BUCKET
     logger = prefect.context.get("logger")
     logger.info(f"deleting file: {str(key)}")
-    s3 = boto3.resource('s3')
+    s3 = boto3.resource("s3")
     s3.Object(bucket, key).delete()
 
 
-with Flow(FLOW_NAME, executor=LocalDaskExecutor(), schedule=get_interval_schedule(minutes=1440)) as flow:
+@task()
+def ordered_dataset_insert_transform() -> None:
+    """This task will incrementally insert new records into the article_content_ordered_live table.
+    This table was created manually from existing raw data as a starting point.
+    Query is built so that this task is itempotent.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Adding new records to article_content_ordered_live dataset...")
+    with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+        result = conn.execute_string(INSERT_ORDERED_SQL)
+        logger.info(f"Inserted {result[1].rowcount} rows...")
+
+
+@task()
+def weekly_reordering_transform() -> None:
+    """This task will perform a reordering of the article_content_ordered_live table weekly.
+    This involves a new table being swaped with the old.
+    Grants are cleaned up.
+    Logic makes it that this will only run on Saturday if the last_ordered date is older than a week.
+    A single run of this task updates the last _ordered date and prevents multiple runs.
+    """
+    logger = prefect.context.get("logger")
+    logger.info("Checking to see if reordering is needed...")
+    with sf.connect(**SNOWFLAKE_DEFAULT_DICT) as conn:
+        result = (
+            conn.cursor()
+            .execute(
+                f"""select datediff('hours', max(last_ordered), current_timestamp()) > 168 as ready_for_reordering
+   from {DATABASE_NAME}.{SCHEMA_NAME}.article_content_ordered_live;"""
+            )
+            .fetchone()
+        )
```

Comment:
I moved to using the Prefect SQL File Task.  Though it does not offer parameter support and I need the execute_string method it uses.  I think string formatting these 2 queries is fine given that the variables cannot be updated externally in a way that would allow SQL injection.

Reply:
I moved to using the Snowflake SQL File task and the db and schema task args as well.  Thank you for the advice on that.  I totally agree.